src/PSRule.Rules.Azure/rules/Azure.ACR.Rule.ps1

```diff
@@ -57,6 +57,15 @@ Rule 'Azure.ACR.SoftDelete' -Ref 'AZR-000310' -Type 'Microsoft.ContainerRegistry
     $Assert.HasFieldValue($TargetObject, 'properties.policies.softDeletePolicy.retentionDays').Reason($LocalizedData.ACRSoftDeletePolicyRetention, $TargetObject.name)
 }
 
+# Synopsis: Disable export of artifacts from container registries.
+Rule 'Azure.ACR.ExportPolicy' -Ref 'AZR-000403' -Type 'Microsoft.ContainerRegistry/registries' -Tag @{ release = 'GA'; ruleSet = '2024_12'; 'Azure.WAF/pillar' = 'Security'; } -Labels @{ 'Azure.MCSB.v1/control' = @('DP-2') } {
```

Comment:
Regenerate this rule in YAML within the `Azure.ACR.Rule.yaml` file. Also:

- Use the reference ID: `AZR-000495`.
- Use the ruleSet: `2025_09`.

Please explain the reasoning for using `DP-2`.